src/plugins/expressions/public/loader.ts

```diff
@@ -145,7 +145,10 @@ export class ExpressionLoader {
       .getData()
       .pipe(
         delay(0), // delaying until the next tick since we execute the expression in the constructor
-        filter(({ partial }) => params.partial || !partial)
+        filter(({ partial }) => params.partial || !partial),
+        params.partial && params.throttle
+          ? throttleTime(params.throttle, asyncScheduler, { leading: true, trailing: true })
+          : identity
```

Comment:
should we have a default throttle time ?

Reply:
I was thinking of making this behavior optional. If there is a need to throttle the subject, we can pass a non-zero value. Otherwise, it will not be throttled.

Reply:
i am wondering if it should rather be the other way around, so throttled by default, you can turn it off (by setting value to 0) or finetune it

Reply:
I guess that makes sense. I flipped the behavior in the latest commit. Thanks 👍